### DISCUSSION TITLE
python runtime mismatch when trying to deploy a fastAPI app to vercel

### DISCUSSION THREAD

**Discussion by LeeSkies** [vercel/community#3593] [Help]

**Comment by DuncanLHS** [comment]
Vercel only supports Python 3.9
https://vercel.com/docs/functions/serverless-functions/runtimes/python#python-version

**Comment by LeeSkies** [comment]
iv'e found the problem but the answer isn't here, and i cant seem to find the delete post option, if there's a moderator seeing this please delete this post.
thx

**Comment by LeeSkies** [comment]
hey, here's the post I opened in overflow, the solution is in the comments.
https://stackoverflow.com/questions/76893012/python-runtime-mismatch-when-trying-to-deploy-a-fastapi-app-to-vercel
